### ISSUE TITLE
Failing to install with docker method

### ISSUE THREAD

**Issue by abdimussa87** [quran/quran.com-images#49]
I keep getting the below error when I ran `docker-compose up -d`:

```
8.427 W: Failed to fetch http://deb.debian.org/debian/dists/jessie/main/binary-armhf/Packages  404  Not Found
8.427 
8.427 W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/main/binary-armhf/Packages  404  Not Found
8.427 
8.427 E: Some index files failed to download. They have been ignored, or old ones used instead.
failed to solve: process "/bin/sh -c apt-get update -qq &&     apt-get install -y       libgd-gd2-perl libgd-text-perl       libdbd-mysql-perl libdbi-perl       libconfig-yaml-perl       make gcc g++       unzip       curl &&     apt-get clean &&     rm -rf /var/lib/apt/lists/*" did not complete successfully: exit code: 100
```

**Comment by SofiyanIfren** [comment]
I have openned a PR about that : https://github.com/quran/quran.com-images/pull/51

Just use the modified Dockerfile there :

`FROM debian:buster

MAINTAINER Hossam Hammady <github@hammady.net>

RUN apt-get update -qq && \
    apt-get install -y \
      libgd-gd2-perl libgd-text-perl \
      libdbd-mysql-perl libdbi-perl \
      libconfig-yaml-perl \
      make gcc g++ \
      unzip \
      curl && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*

RUN cd / && \
    curl -L -o zopfli.zip https://github.com/google/zopfli/archive/master.zip && \
    unzip zopfli.zip && \
    cd zopfli-master && \
    make zopflipng && \
    cp zopflipng /usr/local/bin/

WORKDIR /app

RUN curl -L https://cpanmin.us | perl - -M https://cpan.metacpan.org -n Mojolicious

ADD config /app/config
ADD lib /app/lib
ADD res /app/res
ADD script /app/script
ADD Makefile.PL /app/Makefile.PL

RUN cd /app && \
    perl Makefile.PL && \
    make && \
    make install

RUN find /app -type f -name '*.pl' -exec sed -i 's/\r$//' {} \;

RUN sed -i 's/localhost/mysql/' /app/config/database.yaml

CMD /app/script/generate.pl help
`

**Comment by ahmedre** [comment]
this should be fixed now in #50 and #51
